packages/slider/src/mwc-slider-base.ts

```diff
@@ -127,6 +131,11 @@ export class SliderBase extends FormElement {
     </div>`;
   }
 
+  disconnectedCallback() {
+    super.disconnectedCallback();
+    this.mdcFoundation.destroy();
```

Comment:
Why destroy the foundation on `disconnectedCallback()`?

Reply:
to remove all the body resize observers

Reply:
Seems like we either need an explicit removeEventListener wrapper in the foundation, or we'll need to recreate the foundation in `connectedCallback()`

Reply:
re-initted in connected callback